gradle-guide-error-prone/src/main/java/com/palantir/gradle/guide/errorprone/UseGradleLoggerInGradlePlugins.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.guide.errorprone;
+
+import com.google.auto.service.AutoService;
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.BugPattern.SeverityLevel;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.matchers.Matcher;
+import com.google.errorprone.matchers.Matchers;
+import com.sun.source.tree.ClassTree;
+import com.sun.source.tree.VariableTree;
+
+@AutoService(BugChecker.class)
+@BugPattern(
+        severity = SeverityLevel.ERROR,
+        summary = "Use Gradle's `Logger` instead of `SafeLogger` within Gradle plugins. "
+                + "Safe logging isn't necessary for Gradle plugins which don't process sensitive data.")
+public final class UseGradleLoggerInGradlePlugins extends GradleGuideBugChecker
```

Comment:
We should also force direct slf4j loggers (ie `private static final org.slf4j.logger.Logger log`) to be the Gradle logging type too. This should be safe to autofix. There are far, far more instances of this than the SafeLogger one, but the safe logger one does cause more issues.